fastchat/protocol/openai_api_protocol.py

```diff
@@ -116,11 +117,11 @@ class EmbeddingsResponse(BaseModel):
 
 class CompletionRequest(BaseModel):
     model: str
-    prompt: str
+    prompt: Union[str, List[Any]]
     suffix: Optional[str] = None
     temperature: Optional[float] = 0.7
     n: Optional[int] = 1
-    max_tokens: Optional[int] = None
+    max_tokens: Optional[int] = 16
```

Comment:
why 16? It is too small

Reply:
Because openai API use 16, see [here](https://platform.openai.com/docs/api-reference/completions/create)